src/public/modules/forms/admin/directiveViews/settings-form.client.view.html

```diff
@@ -410,7 +401,7 @@
               ng-model="tempForm.webhook.url"
               placeholder="https://your-webhook.com/url"
               ng-required="false"
-              ng-disabled="isFormPublic() || doesFormContainAttachments()"
```

Comment:
may still want to block it on the client first, since there is no way to actually consume the attachment URLs if its sent to the webhook

Reply:
^ sorry what do you mean by this?

Reply:
undo this client change? since the formsg sdk still does not have an utility function to decrypt the attachment object? May result in some form admins adding a webhook for their form with attachments that they do not know what to deal with. 

but then again, not like responses are lost; the attachments can still be retrieved from the responses tab. May be a moot point!

Reply:
oh yeah sure, I agree with that. unless we can make the SDK changes before this code is merged. the SDK changes seem pretty straightforward, since we already have all the required utilities for decryption.

Reply:
Yeah we can leave this PR out of the develop branch until the corresponding SDK changes goes in too.